src/hotspot/share/gc/g1/g1ConcurrentMark.hpp

```diff
@@ -225,7 +225,9 @@ class G1CMMarkStack {
 
     size_t capacity() const { return _capacity; }
 
+    // Expand the mark stack doubling its size.
```

Comment:
This one doesn't guarantee "expansion"; maybe `try_expand` to match its sibling.